basis/basis_link

```diff
@@ -13,24 +13,35 @@
  _________________________________________________________________________ 
 | Keyword                           | Filename                            |
 |-------------------------------------------------------------------------|
-| #STO-3G                           | STO-3G.BAS                          |
-| #3-21G                            | 3-21G.BAS                           |
-| #6-31G                            | 6-31G.BAS                           |
-| #6-31G*                           | 6-31GD.BAS                          |
-| #6-31G(d)                         | 6-31GD.BAS                          |
-| #6-31G**                          | 6-31GDP.BAS                         |
-| #6-31G(d,p)                       | 6-31GDP.BAS                         |
-| #6-31G(2df,2pd)                   | 6-31G2DF2PD.BAS                     |
-| #6-311G                           | 6-311G.BAS                          |
-| #6-311G*                          | 6-311GD.BAS                         |
-| #6-311G(d)                        | 6-311GD.BAS                         |
-| #6-311G**                         | 6-311GDP.BAS                        |
-| #6-311G(d,p)                      | 6-311GDP.BAS                        |
-| #6-311G(2df,2pd)                  | 6-311G2DF2PD.BAS                    |
-| #6-311+G(2d,p)                    | 6-311PLG2DP.BAS                     |
-| #cc-pVDZ                          | CC-PVDZ.BAS                         |
-| #def2-svp                         | DEF2-SVP.BAS                        |
-| #def2-sv(p)                       | DEF2-SV_P.BAS                       |
-| #def2-svpd                        | DEF2-SVPD.BAS                       |
+| #STO-3G                           | STO-3G                              |
+| #3-21G                            | 3-21G                               |
+| #6-31G                            | 6-31G                               |
+| #6-31G*                           | 6-31GD                              |
+| #6-31G(d)                         | 6-31GD                              |
+| #6-31GD                           | 6-31GD                              |
+| #6-31G**                          | 6-31GDP                             |
+| #6-31G(d,p)                       | 6-31GDP                             |
+| #6-31GDP                          | 6-31GDP                             |
+| #6-31G(2df,2pd)                   | 6-31G2DF2PD                         |
+| #6-31G2DF2PD                      | 6-31G2DF2PD                         |
```

Comment:
6-31GDP is not a standard basis set name.

Reply:
Also such a change should not be in this PR as it is unrelated to enabling SAD guess read/write.